src/cluster.c

```diff
@@ -4390,6 +4444,83 @@ sds clusterGenNodesDescription(int filter, int use_pport) {
     return ci;
 }
 
+/* Generate a csv-alike representation of the specified cluster node.
+ * See clusterGenNodesDescription() top comment for more information.
+ *
+ * The function returns the string representation as an SDS string. */
+sds clusterGenLinkDescription(clusterLink *link) {
+    serverAssert(link);
+    char events[3], conninfo[CONN_INFO_LEN], *p;
+    sds node_name;
+
+    if (link->node) {
+        node_name = sdsnewlen(link->node->name, CLUSTER_NAMELEN);
+    } else {
+        node_name = sdsnew("unknown");
+    }
+
+    p = events;
+    if (link->conn) {
+        if (connHasReadHandler(link->conn)) *p++ = 'r';
+        if (connHasWriteHandler(link->conn)) *p++ = 'w';
+    }
```

Comment:
I didn't realize this was in client list, I find it hard to imagine it's that useful since I would look at the I/O time more than this. I don't feel strongly about it.